### PR TITLE
Release r1.1 (Release Candidate)

### LINKED ISSUE #164: Generate Release Candidate Carrier Billing versions for Meta Release v0.4 - Fall24
**Problem description**
Generate Release candidate Carrier Billing versions for:
- Carrier Billing Payment: v0.3.0-rc.1
- Carrier Billing Refund: v0.1.0-rc.1

**Expected action**
Generate PR with expected actions as per Release Management and Commonalities Guidelines  

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by PedroDiez**
#### What type of PR is this?

Add one of the following kinds:
* documentation
* subproject management

#### What this PR does / why we need it:

Generation of release r1.1 (release candidate) for Carrier Billing APIs Family

This release candidate r1.1 contains the definition and documentation of
* Carrier Billing v0.3.0-rc.1
* Carrier Billing Refund v0.1.0-rc.1

The API definition(s) are based on
* Commonalities v0.4.0
* Identity and Consent Management v0.2.0

#### Which issue(s) this PR fixes:

Fixes #164

#### Special notes for reviewers:

* Follow Up of #165. Closed as i was not able to make more commits as protected branch
* Follow Guideline Recommendations provided by Rafal
* Adapt recent updates in "# Authorization and authentication" section from I&CM - https://github.com/camaraproject/IdentityAndConsentManagement/pull/186
* Remove "termsOfService" and "contact" from OAS info object

#### Changelog input

```
 Carrier Billing Release r1.1

```

#### Additional documentation 

N/A

### CODE REVIEW

**Review by bigludo7 — DISMISSED** [review]
LGTM

File: README.md
Comment (rartych) [inline README.md:30]: We can change the link to cover branches or tags :)  Small wording correction.
```suggestion
* The release **r1.1 - rc** is available in the [here](https://github.com/camaraproject/CarrierBillingCheckOut/tree/r1.1), and includes the following APIs:
```

**Review by bigludo7 — DISMISSED** [review]

**Review by rartych — APPROVED** [review]
LGTM

**Comment by hdamker** [comment]
@PedroDiez @rartych The PR seems to be in quite good shape. Is there anything open to be done before the release candidate can be created? Also #166 seem to be ready?

I have proactively created https://github.com/camaraproject/ReleaseManagement/issues/70 for the review through Release Management, but please add @camaraproject/release-management_maintainers as reviewer if you want to get the review and approval by them.

**Comment by PedroDiez** [comment]
> @PedroDiez @rartych The PR seems to be in quite good shape. Is there anything open to be done before the release candidate can be created? Also #166 seem to be ready?
> 
> I have proactively created [camaraproject/ReleaseManagement#70](https://github.com/camaraproject/ReleaseManagement/issues/70) for the review through Release Management, but please add @camaraproject/release-management_maintainers as reviewer if you want to get the review and approval by them.


Hi @hdamker! Yes it is ready to be merged. 
Anycase let's follow the "formal" procedure. Have been added RM maintainers to approve it. Indeed it was a doubt that I was to query in today's CB meeting, 100% clear with your comment. Many thanks for helping on this!

**Comment by rartych** [comment]
To be formally correct and following the note below the table:

> "tbd" is in case an alpha or release-candidate API version does not yet provide all mandatory assets for the release

the 7th row of Readiness checklists in RC.1 needs to be changed to:

| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
|----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----:|
|7 | Basic API test cases & documentation | O | M | M | M | **tbd** | link|

**Review by rartych — APPROVED** [review]
Release PR approved - on behalf of Release Management

File: README.md
Comment (hdamker) [inline README.md:30]:
```suggestion
* The release **r1.1 - rc** is available in [r1.1](https://github.com/camaraproject/CarrierBillingCheckOut/tree/r1.1), and includes the following APIs:
```

Comment (hdamker) [reply README.md:30]: Alternative just: 
```suggestion
* The release r1.1 contains the following API definitions (with inline documentation):
```

**Comment by hdamker** [comment]
One small final point spotted ... will support with the merge afterwards.

**Comment by PedroDiez** [comment]
> @PedroDiez @rartych One small final point spotted ... will support with the merge afterwards.

Thanks for the review Herbert

**Review by rartych — APPROVED** [review]

**Comment by hdamker** [comment]
Merging on behalf of Codeowners

**Comment by PedroDiez** [comment]
Many thanks @hdamker!

**Pull request merged**
